dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -373,28 +409,48 @@ export default function CollectCardPaymentScreen() {
       return;
     }
 
-    addLogs({
-      name: 'Confirm Payment Intent',
-      events: [
-        {
-          name: 'Processed',
-          description: 'terminal.confirmPaymentIntent',
-          metadata: {
-            paymententIntentId: paymentIntent.id,
-            chargeId: paymentIntent.charges[0].id,
+    if (paymentIntent.id) {
+      addLogs({
+        name: 'Confirm Payment Intent',
+        events: [
+          {
+            name: 'Processed',
+            description: 'terminal.confirmPaymentIntent',
+            metadata: {
+              paymententIntentId: paymentIntent.id,
+              chargeId: paymentIntent.charges[0].id,
+            },
           },
-        },
-      ],
-    });
+        ],
+      });
+    } else {
+      addLogs({
+        name: 'Confirm Payment Intent',
+        events: [
+          {
+            name: 'Processed',
+            description: 'terminal.confirmPaymentIntent',
+            metadata: {
+              paymententIntentId: '',
+              chargeId: '',
+            },
+          },
+        ],
+      });
+    }
```

Comment:
Can we change "Processed" to "Confirmed" and also combine these two branches into:

``` javascript
      addLogs({
        name: 'Confirm Payment Intent',
        events: [
          {
            name: 'Confirmed',
            description: 'terminal.confirmPaymentIntent',
            metadata: {
              paymententIntentId: paymentIntent.id ? paymentIntent.id : 'null',
              chargeId: paymentIntent?.charges[0]?.id ? paymentIntent.charges[0].id : 'null',
            },
          },
        },
      ],
    });
```